tests/e2e/models/BUILD.bazel

```diff
@@ -52,19 +55,19 @@ iree_lit_test_suite(
     ],
 )
 
-# Larger tests requiring tags skipping them in slow configurations such as
-# sanitizers.
 iree_lit_test_suite(
-    name = "lit_large",
+    name = "lit_enormous",
+    # These tests are too large for even moderately slow configurations
+    # (sanitizers...).
+    size = "enormous",
     srcs = ["resnet50_fake_weights.mlir"],
```

Comment:
This is a pretty clear case where a test size makes sense - the test is running a complicated workload. I'm skeptical about the focus on sanitizers though (we'd still want this tagged as large even if we never ran this test using sanitizers or under an emulator).

Reply:
ok, dropping these comments.